watchtower-plugin/src/main.rs

```diff
@@ -466,18 +484,25 @@ async fn on_commitment_revocation(
         } else {
             if status.is_subscription_error() {
                 log::warn!(
-                    "There is a subscription issue with {}. Adding appointment to pending",
+                    "There is a subscription issue with {}. Adding {} to pending",
                     tower_id,
+                    appointment.locator
                 );
             } else {
-                log::warn!("{} is {}. Adding appointment to pending", tower_id, status);
+                log::warn!(
+                    "{} is {}. Adding {} to pending",
+                    tower_id,
+                    status,
+                    appointment.locator,
+                );
             }
 
-            plugin
-                .state()
-                .lock()
-                .unwrap()
-                .add_pending_appointment(tower_id, &appointment);
+            let mut state = plugin.state().lock().unwrap();
+            state.add_pending_appointment(tower_id, &appointment);
+            state
+                .unreachable_towers
+                .send((tower_id, appointment.locator))
+                .unwrap();
```

Comment:
I missed that one before, but, we should not send an unreachable tower (not temporarily) to the retrier, right?

Reply:
Yeah, I guess we shouldn't. That also hits subscription errors btw.

So we should add them to pending (db) but not send them to the retrier if they are not temporarily unreachable

Reply:
Fixed it in [56ec2eb](https://github.com/talaia-labs/rust-teos/pull/89/commits/56ec2eb8be0cb90d99f7f091fdbd59f3dcc3ea7b) alongside other minor things regarding the status. The only one worth mentioning is splitting `is_unreachable` in two: `is_unreachable` and `is_temporary_unreachable` given we have a good use case for it now.